src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,8 @@
-const booleanToWord = boolean => {};
-
+const booleanToWord = boolean => {;
```

Comment:
Stray semi colon there!
```suggestion
const booleanToWord = boolean => {
```